en/docs/develop-components/deploy-an-application-with-buildpacks.md

```diff
@@ -249,4 +249,13 @@ Follow the guidelines below based on your language:
     e.g.,
 
     `web: java -jar target/sample.war` <br>
-`
+
+### Build Time Environment Variables
+
+You can configure the environment variables that are necessary to build the component using the Build Configurations Editor in the Build Tab. This is an optional option and not available for Ballerina and WSO2 MI.
+
+These environment variables and values will be passed to buildpack during the build time. Hence, this option can be used to configure buildpack specific environment variables or enviromemt variables required for component build.
+
+e.g.,
+
+- You can override the Maven command of `Java` buildpack using `GOOGLE_MAVEN_BUILD_ARGS` as environment key and `clean install` as value. Refer the following [documentation](https://cloud.google.com/docs/buildpacks/service-specific-configs) for more usages
```

Comment:
```suggestion
For example, you can override the Maven command of `Java` buildpack using `GOOGLE_MAVEN_BUILD_ARGS` as the environment key and `clean install` as the value. 

Refer to the [documentation](https://cloud.google.com/docs/buildpacks/service-specific-configs) for more usages.
```

I think we can separate the external documentation like this

Reply:
Fixed with https://github.com/wso2/docs-choreo-dev/pull/1506/commits/276829871a1bc1bc09226a8d614bf01e15e70d83